core/model/src/main/java/org/eclipse/rdf4j/TransactionSetting.java

```diff
@@ -0,0 +1,9 @@
+package org.eclipse.rdf4j;
```

Comment:
Can you move this to a new package `org.eclipse.rdf4j.common.transaction`? That's really where `IsolationLevel` should also be, but at least let's not add more inferfaces in a place we know we'll want to refactor later. 

Also means that it's probably better to move it to the rdf4j-util module btw, because that's where all the other `org.eclipse.rdf4j.common` packages are.